examples/Qt/coffee/coffee_manager.cpp

```diff
@@ -0,0 +1,142 @@
+#include "coffee_manager.hpp"
+#include <iostream>
+
+CoffeeMachineManager::CoffeeMachineManager(QObject *parent)
+    : QObject{parent}
+{
+    auto app = realm::App("MY_APP_ID");
+    mUser = app.login(realm::App::credentials::anonymous()).get_future().get();
```

Comment:
Is there no way to do this asynchronously?

Reply:
Yeah we could, I'd just have to create a loading view